src/argilla/client/feedback/training/base.py

```diff
@@ -69,6 +72,7 @@ def __init__(
         self._task = task
         self._seed = seed  # split is used for train-test-split and should therefore be fixed
         self._model = model
+        self._tokenizer = tokenizer
```

Comment:
maybe we should mention if the model and tokenizer passed are not being used in case of other frameworks?

Reply:
Good idea